src/main/java/com/algorand/algosdk/transaction/Transaction.java

```diff
@@ -32,7 +32,7 @@
     @JsonProperty("snd")
     public Address sender = new Address();
     @JsonProperty("fee")
-    public BigInteger fee = Account.MIN_TX_FEE_UALGOS;
+    public BigInteger fee = BigInteger.valueOf(0);
```

Comment:
nit: you could use `BigInteger.ZERO` instead of this constant.

Reply:
Change the default constructor to `public Transaction() {}`, it no longer needs to override the fee after this change.